src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/CreateVMFooter.tsx

```diff
@@ -40,8 +41,10 @@ const CreateVMFooter: FC<CreateVMFooterProps> = ({
   sshSecretCredentials,
 }) => {
   const { t } = useKubevirtTranslation();
+  const { tabsData } = useWizardVMContext();
+
   const history = useHistory();
-  const [startVM, setStartVM] = useState(true);
+  const [startVM, setStartVM] = useState(tabsData?.startVM);
```

Comment:
the instance types flow and the templates flow is not the same and you should not use the Wizard context here as it's not in the wizard :) 
please revert changes from this file